content/blog/release-1-2-1.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Rivet 1.2.1 patch release
+description: Rivet 1.2.1 is now available. This patch release fixes a bug in the modal JavaScript where using a modal without a corresponding trigger would throw errors.
+date: "2019-01-31"
+notificationOnly: true
+external_url: https://github.com/indiana-university/rivet-source/releases/tag/v1.2.1
```

Comment:
This is small, but I _think_ we've used camelCase for front matter variables everywhere else. I'm not sure if there's a recommended style for YAML (I'd be fine with standardizing on that), but for now I'd say let's keep it consistent throughout our code base 👍